docs/preflight-in-cluster.md

```diff
@@ -0,0 +1,74 @@
+# Run Preflight In-Cluster
+
+This doc explains how to run Preflight inside a GKE cluster, and get periodic reports in a Google Cloud Storage Bucket.
+
+## Build and push a Preflight image with your packages
+
+This is as easy as running:
+
+```
+export IMAGE_NAME=<my image>
+make build-docker-image
+make push-docker-image
+```
```

Comment:
We should get some images published at some point soon to to avoid this 😀

Reply:
Yes, we need to build images and binaries from Prow. Not done yet.